coordinator/src/admin.rs

```diff
@@ -175,3 +175,24 @@ pub async fn sign_message(
 
     Ok(Json(signature))
 }
+
+pub async fn connect_to_peer(
+    State(state): State<Arc<AppState>>,
+    target: Json<NodeInfo>,
+) -> Result<(), AppError> {
+    let target = target.0;
+    state.node.inner.connect(target).await.map_err(|err| {
+        AppError::InternalServerError(format!("Could not connection to {target}. Error: {err}"))
+    })?;
+    Ok(())
+}
+
+pub async fn is_connected(
+    State(state): State<Arc<AppState>>,
+    Path(target_pubkey): Path<String>,
+) -> Result<Json<bool>, AppError> {
+    let target = target_pubkey.parse().map_err(|err| {
+        AppError::InternalServerError(format!("Invalid public key {target_pubkey}. Error: {err}"))
```

Comment:
Nit: since it's an invalid user Input it should be a clientSide error, e.g. 4xx something